c2corg_api/tests/views/test_route.py

```diff
@@ -57,6 +57,9 @@ def test_post_non_whitelisted_attribute(self):
         }
         self.post_non_whitelisted_attribute(body)
 
+    def test_post_missing_content_type(self):
```

Comment:
You are right, this test could also be in `BaseTestRest`. But actually I would prefer to also list it in the children classes. So that by looking at `TestRouteRest` you directly see which tests are run for the routes.